docs/core-concepts/core-sdk/cli/overview.md

```diff
@@ -0,0 +1,121 @@
+use the native installers to install the CLI or use the installation shell script.
+
+The native installers are primarily meant for developer's machines. The CLI is distributed using each supported platform's 
+native install mechanism, for instance DEB packages on Ubuntu or MSI bundles on Windows. These installers will install 
+and set up the environment as needed for the user to use the CLI immediately after the install. However, they also 
+require administrative privileges on the machine. You can view the installation instructions on the
+[getting started page](https://aka.ms/dotnetcoregs).
+
+Install scripts, on the other hand, do not require administrative privileges. However, they will also not install any 
+prerequisites on the machine; you need to install all of the pre-requisites manually. The scripts are meant mostly for 
+setting up build servers or when you wish to install the tools without admin privileges (do note the pre-requisites 
+caveat above). You can find more information on the [install script reference topic](dotnet-install-script.md). If you are 
+interested in how to set up CLI on your CI build server you can take a look at the [CLI with CI servers](using-ci-with-cli.md) 
+document. 
+
+By default, the CLI will install in a "side-by-side" (SxS) manner. This means that multiple versions of the CLI tools 
+can coexist at any given time on a single machine. How the correct version gets used is explained in more detail in 
+the [driver section](#driver) below. 
+
+### What commands come in the box?
+The following commands are installed by default:
+
+* [new](dotnet-new.md)
+* [restore](dotnet-restore.md)
+* [run](dotnet-run.md)
+* [build](dotnet-build.md)
+* [test](dotnet-test.md)
+* [publish](dotnet-publish.md)
+* [pack](dotnet-pack.md)
+
+Of course, there is a way to import more commands on a per-project basis as well as to add your commands. This is 
+explained in greater detail in the [extensibility section](#extensibility). 
+
+## Working with the CLI
+
+### A short sample
+Before we go into any more details, let's see how working with the CLI looks like from a 10,000-foot view. 
+The sample below utilizes several commands from the CLI standard install to initialize a new simple console application, 
+restore the dependencies, build the application and then run it. 
+
+```console
+dotnet new
+dotnet restore
+dotnet build --output /stuff
+dotnet /stuff/new.dll
+```
+
+### How does it work?
+As we saw in the short sample [above](#a-short-sample), there is a pattern in the way you use the CLI tools. Within that pattern, we can 
+identify three main pieces of each command:
+
+1. The driver ("dotnet")
+2. The command, or "verb"
+3. Command arguments
+
+Let's dig into more details on each of the above.
+
+### Driver
+The driver is named `dotnet`. It is the first part of what you invoke . The driver has two responsibilities:
```

Comment:
Perhaps change the sentence "The driver has two responsibilities:" to something like "The driver can perform one of the following operations depending on what's specified on the command line:". Then you can also remove the following sentence from below "Which of the two things it does is dependent on what is specified on the command line." which is hard to read.